.prettierrc.json

```diff
@@ -1,6 +1,6 @@
 {
   "bracketSpacing": false,
-  "printWidth": 100,
+  "printWidth": 80,
```

Comment:
I think we should do this in a separate PR.

This is difficult to review since there are so many formatting changes.

Reply:
Sounds good!

Reply:
Sounds good!